src/main/java/com/EighthLight/app/Ai.java

```diff
@@ -96,6 +96,25 @@ public void easyMode(IBoard board) {
         }
     }
 
+    private void hardMode(IBoard board) {
```

Comment:
Moved hard mode logic out of make move method.
This logic and negaMax may move to their own HardMode class in the near future.